moveit_core/profiler/include/moveit/profiler/profiler.h

```diff
@@ -83,7 +83,7 @@ class Profiler : private boost::noncopyable
       prof_.begin(name);
     }
 
-    ~ScopedBlock(void)
+    ~ScopedBlock()
```

Comment:
[observation] Did we used to have to compile this using c99?  Seems like a weird artifact.